### PR TITLE
Br review

### PR DESCRIPTION
**Pull request by BenRobertson23**
Made some additions and modifications.

### CODE REVIEW

File: source/includes/api/_api.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ Below are the metrics that the Emotion AI API can analyse and the units that the
 
 |Metric Name|Unit|
 |-----------|----|
-|heartrate|bpm |
+|<a href = "http://help.sensum.co/knowledge_base/topics/why-measure-heart-rate">heartrate</a>|bpm|
 |breathingrate|bpm|
 |temperature|<sup>o</sup>C, assumed to be ambient/external|
 |skintemperature|<sup>o</sup>C|
-|gsr| Siemens<sup>*</sup>|
```
Comment (andrewbolster) [inline source/includes/api/_api.md:46]: NO! The metric values are case sensitive. `gsr`

File: source/includes/ios/_ios.md [hunk 2]
```diff
@@ -1,8 +1,39 @@
...
+* The iOS SDK supports connecting to BLE devices for reading heart rate measurements. For a list of tested compatible devices please view the <a href = "http://help.sensum.co/knowledge_base/topics/what-type-of-sensors-can-i-use"> list of compatible devices</a> at our Knowledge Centre.
+
+**Note:** This document is regularly updated with new devices. Please contact us for integration details. GSR data is only accessible from Shimmer devices at present.
+
+## Accepted Biometric Data Inputs
+
+|Metric Name|Unit|
+|-----------|----|
+|<a href = "http://help.sensum.co/knowledge_base/topics/why-measure-heart-rate">Heart Rate</a>|bpm|
+|<a href = "http://help.sensum.co/knowledge_base/topics/what-is-gsr">GSR</a>|Siemens<sup>*</sup>|
```
Comment (andrewbolster) [inline source/includes/ios/_ios.md]: IOS Doesn't do GSR, so *either* this and the android section could be deduplicated and cross linked *or* this GSR gets cut out.

File: source/includes/ios/_ios.md [hunk 3]
```diff
@@ -227,6 +258,9 @@ Set the frequency of the updates from the accelerometer.
 
 Below are all available commands relating to Bluetooth within the **SensumSDK**.
 
+* **SensumKit** supports connecting to BLE devices for reading heart rate measurements. For a list of tested compatible devices please view the <a href = "http://help.sensum.co/knowledge_base/topics/what-type-of-sensors-can-i-use"> list of compatible devices</a> at our Knowledge Centre.
```
Comment (andrewbolster) [inline source/includes/ios/_ios.md:256]: Might be worth clarifying the relationship between **SensumSDK** and **SensumKit**, i.e. that Kit is specific to iOS

**Review by andrewbolster — CHANGES_REQUESTED** [review]
Looks good but needs a few adjustments.

File: source/includes/api/_api.md (hunk 1 above)
Comment (BenRobertson23) [reply source/includes/api/_api.md:46]: Fixed.

File: source/includes/ios/_ios.md (hunk 3 above)
Comment (BenRobertson23) [reply source/includes/ios/_ios.md:256]: Added a Glossary of Terms to clarify these.

**Comment by BenRobertson23** [comment]
Made these changes plus a few others to better fit with the current naming conventions. Plus I removed the redundant reference to the Android service constants in the index.html.md file.

**Review by andrewbolster — APPROVED** [review]
:+1: Might need to review the use of "GPS" instead of "Location" but that's a wider discussion.

**Pull request merged**